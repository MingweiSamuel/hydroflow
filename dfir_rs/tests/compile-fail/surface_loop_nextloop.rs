fn main() {
    let mut df = hydroflow::hydroflow_syntax! {
        source_iter(0..10) -> next_loop() -> null();
    };
    df.run_available();
}
