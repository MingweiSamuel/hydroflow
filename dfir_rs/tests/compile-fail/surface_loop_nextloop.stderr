error[E0433]: failed to resolve: use of undeclared crate or module `hydroflow`
 --> tests/compile-fail/surface_loop_nextloop.rs:2:18
  |
2 |     let mut df = hydroflow::hydroflow_syntax! {
  |                  ^^^^^^^^^ use of undeclared crate or module `hydroflow`
